python/tvm/tir/tensor_intrin/__init__.py

```diff
@@ -16,4 +16,8 @@
 # under the License.
 # pylint: disable=unused-import
 """Intrinsics for tensorization."""
-from . import arm_cpu, cuda, rocm, x86, hexagon
+import tvm
```

Comment:
One more small nit,
Let's relax a bit the dependency in this hotpath with: ```from tvm.runtime import enable```
Like here (tir init): https://github.com/apache/tvm/blob/main/python/tvm/tir/__init__.py#L20